e2e/basicProps.e2e.ts

```diff
@@ -0,0 +1,55 @@
+/* eslint-env detox/detox, jest/globals */
+import {
+  afterAllTestPlan,
+  beforeAllTestPlan,
+  waitForClearLogs,
+  waitForTestPlan,
+  waitForLogLabel,
+  waitForTap,
+} from './testPlan';
+
+describe('Basic Props', () => {
+  beforeAll(beforeAllTestPlan);
+  afterAll(afterAllTestPlan);
+
+  it('paused controls playback state', async () => {
+    await waitForTestPlan(`
+    inputs:
+    - paused: true
+    events:
+    - onPlayerStateChange
+    `);
+    await waitForTap(by.id('paused'));
+    await waitForLogLabel('onPlayerStateChange ::: Playing');
```

Comment:
In a similar way, a DSL around validating outputs could also be useful. I think having separate test setup, execution and validation snippets can be useful, which I think it where this is going.

Ideally we're not verifying directly via logs, but collected output with post-execution validation (along with a more defined interface for the collected output) could work well.